### ISSUE TITLE
Mirror Room

### ISSUE THREAD

**Issue by TiTiMartos** [MontyTRC89/Tomb-Editor#918]
Mirror Rooms, for when?

**Comment by Stranger1992** [comment]
> Mirror Rooms, for when?

Hi. Can you please clarify what you are wanting or else I will close this issue.

Thanks

**Comment by TiTiMartos** [comment]
Hello, in TRLE and NGLE it is possible to create mirrors using the mirror command, is it possible in TEN?

**Comment by Stranger1992** [comment]
Hi

Not at this time,no. It is something the development team will be looking
into and is very much wanted.

**Comment by TiTiMartos** [comment]
Thank you, and sorry for my poor English. 
I will be waiting for news on the subject. Thanks.

**Comment by illyaine** [comment]
Yes, I would also be very happy about the mirror option. Here, of course, the best way is to improve it directly. That everything mirrors itself, not like in the TRNG.

On the floor, ceiling, walls.
Or only on a specific object. Whatever is possible.
